redbot/cogs/mod/kickban.py

```diff
@@ -218,7 +232,22 @@ class KickBanMixin(MixinMeta):
 
         If days is not a number, it's treated as the first word of the reason.
         Minimum 0 days, maximum 7. Defaults to 0."""
-
+        author = ctx.author
+        guild = ctx.guild
+        if reason == None:
+            reason = "No reason was given."
+        else:
+            reason = reason
+        self == self.bot.get_cog("Mod")
```

Comment:
Same thing as above, if its supposed to be a `=`, then it isn't needed as `self` is already the Mod cog.